src/main.js

```diff
@@ -105,7 +121,7 @@ function makeCoverButton() {
 }
 
 function saveCover() {
-  if (!savedCovers.includes(currentCover)) {
+  if (!savedCoverIDs.includes(currentCover.id)) {
```

Comment:
I changed this to `.id` since it looked to be having a problem with the truthiness of our `currentCover` object...